lib/rules/one-dependency-per-line.js

```diff
@@ -77,16 +83,71 @@ module.exports = function (context) {
         return settings[setting] === "never";
     }
 
+    function getFileIndentationLevel() {
```

Comment:
it might be cleaner to pass in an argument here rather than rely on a module-level `args`